requirements.txt

```diff
@@ -4,36 +4,37 @@
 #
-argcomplete==1.11.1       # via -r requirements.in (line 1)
-boto3==1.14.2             # via -r requirements.in (line 2), kappa
-botocore==1.17.2          # via boto3, s3transfer
+argcomplete==1.11.1       # via -r requirements.in
+boto3==1.16.46            # via -r requirements.in, kappa
+botocore==1.19.46         # via boto3, s3transfer
 certifi==2020.4.5.2       # via requests
 cfn-flip==1.2.3           # via troposphere
 chardet==3.0.4            # via requests
 click==7.1.2              # via cfn-flip, kappa, pip-tools
-docutils==0.15.2          # via botocore
-durationpy==0.5           # via -r requirements.in (line 3)
-future==0.18.2            # via -r requirements.in (line 11)
-hjson==3.0.1              # via -r requirements.in (line 4)
+durationpy==0.5           # via -r requirements.in
+future==0.18.2            # via -r requirements.in
+hjson==3.0.1              # via -r requirements.in
 idna==2.9                 # via requests
-jmespath==0.10.0          # via -r requirements.in (line 5), boto3, botocore
-kappa==0.6.0              # via -r requirements.in (line 6)
-pip-tools==5.2.1          # via -r requirements.in (line 22)
+importlib-metadata==1.7.0  # via argcomplete
+jmespath==0.10.0          # via -r requirements.in, boto3, botocore
+kappa==0.6.0              # via -r requirements.in
+pip-tools==5.2.1          # via -r requirements.in
 placebo==0.9.0            # via kappa
-python-dateutil==2.8.1    # via -r requirements.in (line 8), botocore
-python-slugify==4.0.0     # via -r requirements.in (line 9)
-pyyaml==5.3.1             # via -r requirements.in (line 10), cfn-flip, kappa
-requests==2.23.0          # via -r requirements.in (line 13)
+python-dateutil==2.8.1    # via -r requirements.in, botocore
+python-slugify==4.0.0     # via -r requirements.in
+pyyaml==5.3.1             # via -r requirements.in, cfn-flip, kappa
+requests==2.23.0          # via -r requirements.in
 s3transfer==0.3.3         # via boto3
-six==1.15.0               # via -r requirements.in (line 14), cfn-flip, pip-tools, python-dateutil
+six==1.15.0               # via -r requirements.in, cfn-flip, pip-tools, python-dateutil
 text-unidecode==1.3       # via python-slugify
-toml==0.10.1              # via -r requirements.in (line 15)
-tqdm==4.46.1              # via -r requirements.in (line 16)
-troposphere==2.6.1        # via -r requirements.in (line 17)
+toml==0.10.1              # via -r requirements.in
+tqdm==4.46.1              # via -r requirements.in
+troposphere==2.6.1        # via -r requirements.in
 urllib3==1.25.9           # via botocore, requests
-werkzeug==0.16.1          # via -r requirements.in (line 19)
-wheel==0.34.2             # via -r requirements.in (line 20)
-wsgi-request-logger==0.4.6  # via -r requirements.in (line 21)
+werkzeug==0.16.1          # via -r requirements.in
+wheel==0.34.2             # via -r requirements.in
+wsgi-request-logger==0.4.6  # via -r requirements.in
+zipp==3.4.0               # via importlib-metadata
```

Comment:
the diff is a bit hard to read, here's what actually changed:

**updated dependencies:**
```
boto3==1.16.46            # via -r requirements.in, kappa
botocore==1.19.46         # via boto3, s3transfer
```

**new dependencies:**

```
importlib-metadata==1.7.0  # via argcomplete
zipp==3.4.0               # via importlib-metadata
```

Reply:
1.16.45 in one requirements file, but 1.16.46 in another?

Reply:
I've updated the `requirements.in` so it is also 1.16.46, not sure why the inconsistency arose